oak/src/oak/server.cpp

```diff
@@ -175,75 +112,207 @@ class Request : public cloe::Request {
 
  public:
   /**
-   * Create a new Request from ServerImpl.
-   *
-   * - This should not panic.
+   * Creates a new Request from a http_connection
```

Comment:
I'm torn on this one. I understand that a lot of documentation in the C++ standard libarary uses this descriptive tense, but in our project we've been trying to use the imperative, I think. It's not a big deal, but it's nice to be consistent.